libs/agno/agno/workflow/workflow.py

```diff
@@ -176,6 +181,7 @@ def __init__(
         description: Optional[str] = None,
         db: Optional[BaseDb] = None,
         steps: Optional[WorkflowSteps] = None,
+        agent: Optional["WorkflowAgent"] = None,  # type: ignore
```

Comment:
If you import correctly in the WorkflowAgent class you should be ok to avoid a circular import